docs/client/getting-started.mdx

```diff
@@ -88,97 +121,105 @@ const getEVMSigner = async () => {
 }
 ```
 
-```ts SVM Signer
+```ts Svm Signer
 // For Solana transactions, we'll use the Phantom wallet adapter
 // npm install @solana/wallet-adapter-phantom
 import { PhantomWalletAdapter } from '@solana/wallet-adapter-phantom';
 
-const getSVMSigner = async () => {
+const getSvmSigner = async () => {
   const phantom = new PhantomWalletAdapter();
   await phantom.connect();
   return phantom;
 };
 ```
 </CodeGroup>
 
-### Finalizing the Client Initialization
-Now, pass the signers and your optional configuration options into your `SkipClient` instance:
-
-```ts
-const skipClient = new SkipClient({
-  getCosmosSigner,
-  getEVMSigner,
-  getSVMSigner,
-  // ...configOptions,
-});
-```
 </Step> <Step title="Query Basic Info">
 
-With your client initialized, you can query balances, supported chains and assets.
+With the library initialized, you can query balances, supported chains and assets using the imported functions.
+
+### Query Examples
 
 <CodeGroup>
 
-```ts List of Supported Chains
+```ts Supported Chains
+import { chains } from "@skip-go/client";
+
 // returns a Chain[] of all supported Cosmos mainnet chains
-const cosmosChains = await skipClient.chains();
+const cosmosChains = await chains();
 
 // include EVM and SVM chains
-const allChains = await skipClient.chains({
-  includeEVM: true,
-  includeSVM: true,
+const allChains = await chains({
+  includeEvm: true,
+  includeSvm: true,
 });
 
 // only show testnet chains
-const testnetChains = await skipClient.chains({
+const testnetChains = await chains({
   onlyTestnets: true
 });
 ```
 
-```ts Map of Supported Assets
-// returns `Record<string, Asset[]>
-const assets = await skipClient.assets({
-  includeEvmAssets: true,
-  includeSvmAssets: true,
-});
+```ts Supported Assets
+import { assets } from "@skip-go/client";
+
+// returns `Record<string, Asset[]>`
+const allAssets = await assets(); 
```

Comment:
we could keep this -> `assets({
  includeEvmAssets: true,
  includeSvmAssets: true,
})`